include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -428,10 +428,18 @@ Operator<dim, Number>::setup_operators()
 
 template<int dim, typename Number>
 void
-Operator<dim, Number>::setup_solver()
+Operator<dim, Number>::setup_solver(double const & scaling_factor_mass)
 {
   pcout << std::endl << "Setup elasticity solver ..." << std::endl;
 
+  double active_scaling_factor_mass =
+    param.problem_type == ProblemType::Unsteady ? scaling_factor_mass : 0.0;
+
+  if(param.large_deformation)
+    elasticity_operator_nonlinear.set_scaling_factor_mass_operator(active_scaling_factor_mass);
+  else
+    elasticity_operator_linear.set_scaling_factor_mass_operator(active_scaling_factor_mass);
+
```

Comment:
I assume this is in analogy to how it is done in other modules such as IncNS? I think there is no need to compute an active scaling factor, because it will never be used for a steady problem (because the operator itself contains an if-statement `if(unsteady)`)? If so, I suggest to skip the first two lines.

BTW: The circumstance that a setup of the operator is done in `setup_solvers()` is exactly the Henne-Ei-Problem we have discussed in person.